src/win/conpty.cc

```diff
@@ -223,13 +229,8 @@ static NAN_METHOD(PtyStartProcess) {
   }
 
   Nan::Set(marshal, Nan::New<v8::String>("fd").ToLocalChecked(), Nan::New<v8::Number>(-1));
-  {
-    std::string coninPipeNameStr(inName.begin(), inName.end());
-    Nan::Set(marshal, Nan::New<v8::String>("conin").ToLocalChecked(), Nan::New<v8::String>(coninPipeNameStr).ToLocalChecked());
-
-    std::string conoutPipeNameStr(outName.begin(), outName.end());
-    Nan::Set(marshal, Nan::New<v8::String>("conout").ToLocalChecked(), Nan::New<v8::String>(conoutPipeNameStr).ToLocalChecked());
-  }
+  Nan::Set(marshal, Nan::New<v8::String>("conin").ToLocalChecked(), Nan::New<v8::String>(from_wstring(inName)).ToLocalChecked());
```

Comment:
Similar to above, I'm not sure if there's an efficient way to convert wchar_t to char. Using `from_wstring(inName)` directly seems faster than using `from_wstring(inName)` and then calling `.length()` and `.c_str()` on it.

Reply:
Unresolving this conversation. If we want to use the `(const char*, int)` function instead of the `std::string&` one, We'd need to call `static_cast<int>(strlen(char*))` just to get the length of the `char*`, or add a separate `int* bufferSize` parameter to `from_wstring`. I'm wondering which you prefer in this case?